examples/typescript-with-swc/src/App.tsx

```diff
@@ -13,6 +14,7 @@ function App() {
       <ClassNamed />
       <FunctionDefault />
       <FunctionNamed />
+      <ArrowFunction />
```

Comment:
Is it possible to add this for all examples 🙏🏻 ?

Reply:
Sure, will do